orc8r/cloud/docker/controller/Dockerfile

```diff
@@ -8,7 +8,7 @@ FROM --platform=$PLATFORM ubuntu:xenial as base
 ENV GO111MODULE on
 ENV GOPATH ${USER}/go
 ENV GOBIN /build/bin
-ENV PATH ${PATH}:${GOBIN}:${GOPATH}/bin
+ENV PATH ${PATH}:${GOBIN}:${GOPATH}/bin:/usr/local/go/bin
```

Comment:
Why do we need to add this to the path now? Doesn't L33 take care of this already?

Reply:
`ln -s /usr/local/go/bin/go /usr/local/bin/go` in L33 only creates a symlink in `/usr/local/bin` for the go binary.

Reply:
Okay, but `gofmt` is needed as well. I just don't understand what changed from 1.18 to 1.19 and why we need the path now?

Reply:
I also don't understand how it could work in Go 1.18.

Reply:
I have added an additional symlink for `gofmt` and removed the path from `PATH`.

Reply:
Okay :)